mypy/checkexpr.py

```diff
@@ -2191,7 +2192,8 @@ def visit_backquote_expr(self, e: BackquoteExpr) -> Type:
     def accept(self,
                node: Expression,
                type_context: Type = None,
-               allow_none_return: bool = False
+               allow_none_return: bool = False,
+               disallow_any: bool = True,
```

Comment:
I'd consider naming this `always_allow_any` (or similar) instead.  It's a bit misleading as-is.